### PR TITLE
Fix odometry publishing

### LINKED ISSUE #10: No odometry in simulation
When running Noah gazebo simulation:
1. There is no odom frame published
2. There are no messages published to the odom topic

Having odometry published by the simulation is normally a desired feature and usually required.
Looking at the URDF, I understand that it is intended to be published, but it is not.

### PR DESCRIPTION
**Pull request by tulku**
With these changes we can see the robot position history using the odometry visualization in rviz:
Screenshot from 2022-12-16 17-10-31

I re-indented one of the xacro files because it was hard to edit if not. I tried to minimize the changes to the other one though.

Closes #10

### CODE REVIEW

File: noah_description/urdf/accurate_base_link.urdf.xacro [hunk 1]
```diff
@@ -12,6 +12,20 @@
 
 		<!-- BASE************************************************************************** -->
 
+        <link name="base_footprint">			
+            <!--
+            Dummy initia-less link to prevent the following error:
```
Comment (francocipollone) [inline noah_description/urdf/accurate_base_link.urdf.xacro]: nit: 
```suggestion
            Dummy inertia-less link to prevent the following error:
```

**Review by francocipollone — DISMISSED** [review]
Thanks for the fix. 
LGTM!

**Comment by tulku** [comment]
@francocipollone thanks for the review. Comments addressed!

**Review by francocipollone — APPROVED** [review]
LGTM!

**Pull request merged**